libs/x-content/src/main/java/org/elasticsearch/xcontent/support/MapXContentParser.java

```diff
@@ -28,23 +26,10 @@
  */
 public class MapXContentParser extends AbstractXContentParser {
 
-    private XContentType xContentType;
+    private final XContentType xContentType;
     private TokenIterator iterator;
     private boolean closed;
 
-    public static XContentParser wrapObject(Object sourceMap) throws IOException {
```

Comment:
there's a single user of this left, hence I moved it to a private method local to where it's still needed.